### PR TITLE
Add docs for manually resizing an EBS volume

### PR DESCRIPTION
**Pull request by barrucadu**
[Rendered](https://github.com/alphagov/govuk-developer-docs/blob/msw/resize-ebs/source/manual/manually-resize-ebs.html.md)

### CODE REVIEW

File: source/manual/manually-resize-ebs.html.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+In AWS, [we use Elastic Block Store (EBS)][ebs] to attach additional
+block storage to EC2 instances.
+
+EBS is not the same thing as [Elastic File System (EFS)][efs], which
+is more like a shared NFS volume and does not need to be manually
+resized.
+
+[ebs]: https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AmazonEBS.html
+[efs]: https://docs.aws.amazon.com/efs/latest/ug/whatisefs.html
+[brexit-tool]: https://www.gov.uk/get-ready-brexit-check
```
Comment (AlanGabbianelli) [inline source/manual/manually-resize-ebs.html.md:19]: I don't see this used anywhere
```suggestion
```

Comment (barrucadu) [reply source/manual/manually-resize-ebs.html.md]: The risks of copying pre-existing pages...

**Review by AlanGabbianelli — APPROVED** [review]

File: source/manual/manually-resize-ebs.html.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
+---
+owner_slack: "#re-govuk"
+title: Manually resize EBSs (elastic block store)
+section: Infrastructure
+layout: manual_layout
+parent: "/manual.html"
+last_reviewed_on: 2020-12-01
```
Comment (ChrisBAshton) [inline source/manual/manually-resize-ebs.html.md:7]: You don't need this line.

**Review by ChrisBAshton — COMMENTED** [review]
GitHub had a wobbly yesterday with GitHub Actions. I've tried kicking off a re-run of the tests but it always seems to show me yesterday's logs. You may need to physically push a new commit to get a clean CI build.

**Review by ChrisBAshton — APPROVED** [review]

**Pull request merged**